src/data/__tests__/calculate.test.ts

```diff
@@ -135,36 +135,38 @@ describe('calculate', () => {
     const response = calcValue(data)
     // average * 2 people * outdoor * 1 hr * their mask * your mask
     expect(response).toBeCloseTo(
-      ((((PREVALENCE * 2) / 20) * 0.06 * B117_CONTAGIOUSNESS_ADJUSTMENT) /
-        4 /
-        1) *
+      ((PREVALENCE * 2) / 20) *
+        0.06 *
+        B117_CONTAGIOUSNESS_ADJUSTMENT *
+        0.3333 *
```

Comment:
you could do ` / 3` instead of `* 0.333`. or `* (1/3)`

Reply:
Fixed.